pytorch_lightning/profiler/profilers.py

```diff
@@ -287,9 +295,9 @@ def __del__(self):
             self.output_file.close()
 
 
-class PyTorchProfiler(BaseProfiler):
+class LegacyPyTorchProfiler(BaseProfiler):
```

Comment:
Do we plan to deprecate it?

Reply:
Not for now. It is still used from 1.3 to 1.7. But 1.8 API completely changed.